packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -318,33 +314,18 @@ contract Deploy is Deployer {
         save("ProtocolVersions", address(dso.protocolVersionsImpl()));
     }
 
-    /// @notice Deploy a new OP Chain, with an existing SuperchainConfig provided
-    function setupOpChain() public {
```

Comment:
This diff is really funky, but basically what happened is that `setupOPChain()` was removed and all of its actions were added into `deployOPChain()`.

Reply:
40a51fec450461a20cf2bad0577b2b14fc452c25 might be easier to read.